stakingModel/contracts/SimpleDAIStaking.sol

```diff
@@ -23,7 +24,7 @@ interface cERC20 {
 * or withdraw their stake in DAI
 * the contracts buy cDai and can transfer the daily interest to the owner (DAO)
 */
-contract SimpleDAIStaking is DSMath, Pausable, SchemeGuard {
+contract SimpleDAIStaking is DSMath, Pausable, FeelessScheme, ActivePeriod {
```

Comment:
this contract doesnt transfer any GD and is not part of the DAO so no need for feeless or activeperiod

Reply:
updated 46c730e357a4344e67f8ad6ae2d800c1255ca9bf